### PR TITLE
fixed specification description generation

### PR DESCRIPTION
**Pull request by mpashkovskiy**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -84,12 +84,12 @@ function updateSpecFromPackage() {
     spec.info.license = { name: packageInfo.license };
   }
 
-  spec.info.description = '[Specification JSON](' + packageInfo.baseUrlPath + '/api-spec)';
-  if (packageInfo.baseUrlPath) {
+  packageInfo.baseUrlPath = packageInfo.baseUrlPath || '';
+  const baseSpecPath = packageInfo.baseUrlPath + '/api-spec';
+  spec.info.description = `Specification JSONs: [v2](${baseSpecPath}/v2), [v3](${baseSpecPath}/v3).`;
```
Comment (maxiejbe) [inline index.js:89]: Would you use here `versions.OPEN_API_V2` + `versions.OPEN_API_V3` constants?

File: test/index_tests.js [hunk 2]
```diff
@@ -37,6 +37,8 @@ it('WHEN patch function is provided THEN it is applied to spec', done => {
       request.get(`http://localhost:${port}${path}?a=1`, () => {
         const spec = generator.getSpec();
         expect(spec.info.title).toBe(newTitle);
+        expect(spec.info.description).toContain('(/api-spec/v2)');
```
Comment (maxiejbe) [inline test/index_tests.js:40]: Same here: `versions.OPEN_API_V2` + `versions.OPEN_API_V3`. Think we need to export versions from `lib/openapi.js `on the test file.

File: test/index_tests.js [hunk 3]
```diff
@@ -235,7 +241,9 @@ it('WHEN package json does not include baseUrlPath THEN spec description is not
   setTimeout(() => {
     const spec = generator.getSpec();
     expect(spec.basePath === undefined).toBeTruthy();
-    expect(spec.info.description.indexOf('base url') === -1).toBeTruthy();
+    expect(spec.info.description).toContain('(/api-spec/v2)');
```
Comment (maxiejbe) [inline test/index_tests.js:244]: Same here 👍 `versions.OPEN_API_V2` + `versions.OPEN_API_V3`.

File: index.js [hunk 4]
```diff
@@ -84,12 +84,12 @@ function updateSpecFromPackage() {
...
+  if (packageInfo.baseUrlPath !== '') {
     spec.basePath = packageInfo.baseUrlPath;
```
Comment (maxiejbe) [inline index.js:91]: Does empty `spec.basePath` affect swagger-ui tests execution?

**Review by maxiejbe — COMMENTED** [review]
Looking good overall!

File: index.js (hunk 4 above)
Comment (mpashkovskiy) [reply index.js:92]: it doesn't just wanted to avoid condition in line 88

File: test/index_tests.js (hunk 3 above)
Comment (mpashkovskiy) [reply test/index_tests.js]: fixed everywhere

File: test/index_tests.js (hunk 2 above)
Comment (mpashkovskiy) [reply test/index_tests.js]: fixed

File: index.js (hunk 1 above)
Comment (mpashkovskiy) [reply index.js]: fixed

**Review by maxiejbe — APPROVED** [review]
Just tried it locally, working great!

**Comment by mpashkovskiy** [comment]
version v1.0.27 is published

**Comment by maxiejbe** [comment]
@mpashkovskiy @kiprasmel Working like a charm on example project:

image

Both `/api-spec` and `/api-docs` are generated for v2 + v3.

**Pull request merged**